<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainClass</class>
 <widget class="QMainWindow" name="mainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>669</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>mainClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>401</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLoad">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMesages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>881</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>fhfh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveSpace">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Space</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>РЕЗУЛЬТАТ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineResult">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOperations">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>More operations</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
